tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -6666,10 +6666,14 @@ node_modules/chrome-devtools-frontend/front_end/externs.js(79,17): error TS2551:
 node_modules/chrome-devtools-frontend/front_end/externs.js(82,13): error TS2355: A function whose declared type is neither 'void' nor 'any' must return a value.
 node_modules/chrome-devtools-frontend/front_end/externs.js(86,17): error TS2339: Property 'rotate' does not exist on type 'any[]'.
 node_modules/chrome-devtools-frontend/front_end/externs.js(90,17): error TS2339: Property 'sortNumbers' does not exist on type 'any[]'.
+node_modules/chrome-devtools-frontend/front_end/externs.js(92,13): error TS2304: Cannot find name 'S'.
```

Comment:
That's interesting. These are new? Oh wait. Does this include changes from chrome-devtools-frontend, or is it pinned?

Reply:
Pretty sure it's pinned.

Reply:
![image](https://user-images.githubusercontent.com/2932786/40206949-f0cd1a66-59e6-11e8-9243-a044dc0a273e.png)
Yep, pinned.

Reply:
This is the location of the error, ish:
![image](https://user-images.githubusercontent.com/2932786/40206987-11fc45fe-59e7-11e8-9816-69366eadcfde.png)

Reply:
Looks like @template on generic methods is borked.

Reply:
#24232